src/Graph/GraphFormatter.php

```diff
@@ -250,36 +260,19 @@ public function getGraphLegend() {
 	 *
 	 * @return string
 	 */
-	public static function getWordWrappedText( $text, $charLimit ) {
-		$charLimit = max( [ $charLimit, 1 ] );
-		$segments = [];
-
-		while ( strlen( $text ) > $charLimit ) {
-			// Find the last space in the allowed range.
-			$splitPosition = strrpos( substr( $text, 0, $charLimit ), ' ' );
-
-			if ( $splitPosition === false ) {
-				// If there is no space (lond word), find the next space.
-				$splitPosition = strpos( $text, ' ' );
-
-				if ( $splitPosition === false ) {
-					// If there are no spaces, everything goes on one line.
-					$splitPosition = strlen( $text ) - 1;
+	public function getWordWrappedText( $text, $charLimit ) {
+		return implode( $this->lineSeparator, array_reduce(
+			preg_split( '/\s+/', $text ),
```

Comment:
The code here is a lot simpler now, but as far as I can tell, this line is unused

Reply:
If you mean the function `GraphFormatter::getWordWrappedText()`, it is called in at least two places and is tested.

Reply:
And it could still be made simpler: see the new commit.